netdata-installer.sh

```diff
@@ -26,10 +26,15 @@ if [ "${NETDATA_SOURCE_DIR}" != "${INSTALLER_DIR}" ] && [ "${INSTALLER_DIR}" !=
 fi
 
 # -----------------------------------------------------------------------------
-# Pull in OpenSSL properly if on macOS
-if [ "$(uname -s)" = 'Darwin' ] && [ -d /usr/local/opt/openssl/include ]; then
-  export C_INCLUDE_PATH="/usr/local/opt/openssl/include"
-  export LDFLAGS="-L/usr/local/opt/openssl@1.1/lib"
+# Pull in OpenSSL and other includes and libs properly if on macOS
+if [ "$(uname -s)" = 'Darwin' ]; then
+  if [ "$(uname -m)" = "x86_64" ] && [ -d /usr/local/opt/openssl/include ]; then
+    export C_INCLUDE_PATH="/usr/local/opt/openssl/include"
+    export LDFLAGS="-L/usr/local/opt/openssl/lib"
+  elif [ "$(uname -m)" = "arm64" ] && [ -d /opt/homebrew/opt/openssl/include ]; then
+    export CFLAGS="${CFLAGS} -I/opt/homebrew/include/ -I/opt/homebrew/opt/openssl/include/"
```

Comment:
From what I've read about HomeBrew on Apple Silicon, the situation is even more complicated. The target installation directory is either /opt/homebrew (in case of ARM package/install) or /usr/local/homebrew. It depends on the OS version, Homebrew version, particular package (presumably) and if you use Rosetta or not.
Unfortunately I have no access to M1 mac, so I can't say for sure.

Reply:
Ok, will check for a way to find the path. We also need to check that installing in `/usr/local/` as we have in our docs is ok for various versions...

Reply:
@MrZammler The docs are getting overhauled to not include the `--install` option alongside releaseing the new kickstart script since it doesn’t need it. I’m happy to discuss alternatives to `/usr/local/netdata` for our default install prefix though.

Reply:
So, I tried installing in /usr/local/netdata (requires to run kickstart with sudo, and works). It launched after install and seems to work fine.

But: I got a new netdata user, complete with it's own login. Not sure if this is somethings users would like... I don't know if there is a way to create a user without real login.

And I can't seem to be able to stop it... tried a few combinations for sudo launchctl stop/unload/ etc... and netdatacli.

At this point, if I'm just a casual user checking things out, I would rather not install in /usr/local, without some indication on how to stop/start it, or uninstall.

Reply:
stop/start and the user thing should not have anything to do with `/usr/local`. It sounds like Apple has changed things in some way which is breaking our previously (correctly working) handling of a system user and setup for launchd.